packages/vincent-tools/vincent-tool-erc20-approval/src/lib/vincent-tool.ts

```diff
@@ -0,0 +1,180 @@
+import { z } from 'zod';
+import { createVincentTool } from '@lit-protocol/vincent-tool-sdk';
+import { createPolicyMapFromToolPolicies } from '@lit-protocol/vincent-tool-sdk/src/lib/toolCore/helpers';
+import { createPublicClient, http, parseUnits } from 'viem';
+
+import { getPkpInfo, sendErc20ApprovalTx, getCurrentAllowance } from './tool-helpers';
+import { checkNativeTokenBalance } from './tool-checks';
+
+export const Erc20ApprovalToolParamsSchema = z.object({
```

Comment:
Same as with other tool -- Let's remove the prefixes and export these as generically named properties that shadow the VincentTool API